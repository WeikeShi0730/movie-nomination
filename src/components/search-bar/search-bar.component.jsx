import React from "react";

import "./search-bar.styles.scss";

export const SearchBar = (props) => (
  <input
    className="search-bar"
    type="text"
    placeholder="search movies"
    onChange={props.onSearchChange}
  />
);
