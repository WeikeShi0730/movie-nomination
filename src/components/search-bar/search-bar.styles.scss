.search-bar {
  -webkit-appearance: none;
  border: none;
  border-radius: 25px;
  outline: none;
  padding: 10px;
  width: 50%;
  line-height: 30px;
  margin-top: 30px;
  margin-bottom: 30px;
}

input[type="text"] {
  color: black;
  font-size: 24px;
}
