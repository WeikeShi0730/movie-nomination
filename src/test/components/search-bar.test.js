import React from "react";
import { shallow } from "enzyme";
import { SearchBar } from "../../components/search-bar/search-bar.component";

it("should render SearchBar component", () => {
  expect(shallow(<SearchBar />).getElement()).toMatchSnapshot();
});
