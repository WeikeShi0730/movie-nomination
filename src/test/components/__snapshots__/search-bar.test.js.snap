// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SearchBar component 1`] = `
<input
  className="search-bar"
  placeholder="search movies"
  type="text"
/>
`;
